quickstart.md

```diff
@@ -161,253 +162,53 @@ NAME            HANDLER         AGE
-more details.
-
-With Confidential Containers, the workload container images are never downloaded on the host.
-For verifying that the container image doesn’t exist on the host you should log into the k8s node and ensure the following command returns an empty result:
-```
-root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
-```
-You will run this command again after the container has started.
-
-Create a pod YAML file as previously described (we named it `nginx.yaml`) .
-
-Create the workload:
-```
-kubectl apply -f nginx.yaml
-```
-Output:
-```
-pod/nginx created
-```
+certain platforms may require additional configuration to enable confidential computing. For example, a specific 
+host kernel or firmware may be required. See the [guides](./guides/) for more information.
 
-Ensure the pod was created successfully (in running state):
-```
-kubectl get pods
-```
-Output:
-```
-NAME    READY   STATUS    RESTARTS   AGE
-nginx   1/1     Running   0          3m50s
-```
-
-Now go back to the k8s node and ensure that you still don’t have any bitnami/nginx images on it:
-```
-root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
-```
+## Using CoCo
 
-## Encrypted and/or signed images with attestation
+Below is a brief summary and description of some of the CoCo use cases:
 
-The previous example does not involve any attestation because the workload container isn't signed or encrypted
-and the workload itself does not require any secrets.
+- **Container Launch with Only Memory Encryption (No Attestation)** - Launch a container with memory encryption 
+- **Container Launch with Encrypted Image** - Launch an encrypted container by proving the workload is running 
+  in a TEE in order to retrieve the decryption key
+- **Container Launch with Image Signature Verification** - Launch a container and verify the authenticity and 
+  integrity of an image by proving the workload is running in a TEE
+- **Sealed secret** - Implement wrapped kubernetes secrets that are confidential to the workload owner and are
+  automatically decrypted by proving the workload is running in a TEE
 
-This is not the case for most real workloads. It is recommended to use CoCo with signed and/or encrypted images.
-The workload itself can also request secrets from the attestation agent in the guest.
+The CoCo use cases are implemented differently for each TEE, and they are described in each corresponding 
+[guide](./guides) section. To get started using CoCo without TEE hardware, follow the Non-TEE guide below:
 
-Secrets are provisioned to the guest in conjunction with an attestation, which is based on hardware evidence.
-The rest of this guide focuses on setting up more substantial encrypted/signed workloads using attestation
-and confidential hardware.
+- [Non-TEE](./guides/non-tee.md)
```

Comment:
I think we refer to this as `coco-dev` now. We should probably update that. Also I think we could keep this stuff on the main guide, but I guess it's ok to have a separate page.

Reply:
Updated